src/fidesops/ops/service/connectors/sql_connector.py

```diff
@@ -119,6 +130,7 @@ def retrieve_data(
             return []
         logger.info("Starting data retrieval for %s", node.address)
         with client.connect() as connection:
+            self.set_schema(connection)
```

Comment:
I verified locally that this schema seems to just be set for the duration of the connection until it's closed.  
Before running, the schema is `user.public` when set_schema runs, it becomes `backup_schema`, but once the connection is closed, running `show schema` shows `user.public` again @seanpreston

Reply:
Thanks for taking the time to verify!